2017/proposals/bansal-utkarsh.md

```diff
@@ -0,0 +1,628 @@
+* Port remaining 4 coordinate test files - 4 PRs
+
+### July 31st - August 4th
+* Begin work on **Part 3**. PRs are again worked on in parallel.
+* Port analysis & auxiliary modules - multiple PRs
+
+### August 7th - August 11th
+* Port core & topology modules - multiple PRs
+
+### August 14th - August 18th
+* Port files in the root directory, lib and formats modules - multiple PRs
+
+### August 21st - August 25th, **Final Week**
+* This week is set aside as a buffer to negate any time lost/lag in work. Follow up on reviews and get PRs
+merged into develop.
+
+### August 28th - August 29th, **Submit final work**
+* Submit all the work.
+
+## Future works
+All the points mentioned here are _optional_ and may be done if I'm ahead of schedule
+
+* Benchmarking of tests module.
+* Add support for doctests.
+
+Contributions to MDAnalysis
+===========================
+* Universe now woks with StringIO objects [#1254](https://github.com/MDAnalysis/mdanalysis/pull/1254) (Work in progress)
+* Adds filename attribute to MemoryReader & associated tests [#1252](https://github.com/MDAnalysis/mdanalysis/pull/1252)
+* Port GRO tests to new BaseReader/Writer Test classes [#1196](https://github.com/MDAnalysis/mdanalysis/pull/1196)
+* Add support for start and stop in transfer_to_memory [#1189](https://github.com/MDAnalysis/mdanalysis/pull/1189)
+* Fix error message in case of unidentified file format [#1184](https://github.com/MDAnalysis/mdanalysis/pull/1184)
+* Raises IOError if the toplogy file doesn't exist or can't be accessed [#1177](https://github.com/MDAnalysis/mdanalysis/pull/1177)
+
+
+Contributions to other Open Source Projects
+===========================================
+* https://github.com/mozilla/bedrock/pull/2697
+* https://github.com/mozilla/bedrock/pull/2699
+* https://github.com/mozilla/kuma/pull/3131
+* https://github.com/rrmerugu/django-seed/pull/5
+* https://github.com/rrmerugu/django-seed/pull/4
+* I've also been actively engaged with [@Software-Incubator](https://github.com/Software-Incubator), the software development center of my college where I have
+contributed to various projects which include websites, REST APIs, web based games and iOS applications which are all open
+source.
+* Also built an open source library for django forms [here](https://github.com/utkbansal/crispy-forms-materialize).
+
+
+Personal Information
+===================
+
+**Name**: Utkarsh Bansal
+
+**Email**: bansalutkarsh3@gmail.com
+
+**Github**: utkbansal
+
+**Blog**: http://utkarshbansal.me
+
+**Timezone**: UTC + 5:30
+
+Why are you interested in working with us?
+------------------------------------------
+
+
+Have you used MDAnalysis for your research already?
+--------------------------------------------------
+
+
+Do you have any experience programming?
+----------------------------------------
+
+
+Do you have any exams during GSoC or plan a vacation during the summer?
+----------------------------------------------------------------------
+
```

Comment:
The last 4 questions are still missing. Vacation plans are really important though. We are OK if you take some days off but you have to tell us before hand.